aip/0194.md

```diff
@@ -57,10 +60,12 @@ data being streamed.
 **Bidirectional streaming** request retryability is highly dependent on the type
 of data being streamed.
 
+
 ## Codes that should never be retried
 
 - `OK`: The request succeeded.
 - `CANCELLED`: A user can cancel a request, which we should honour.
```

Comment:
General comment for the document; Is using the first person correct here? Alternatively
```suggestion
- `CANCELLED`: A user can cancel a request, which should be honoured.
```